docs/content/concepts/apis/exporting-apis.md

```diff
@@ -303,7 +311,10 @@ When reconciling exported APIs, controllers usually interact with the API resour
 
 ### Endpoint Slices
 
-An API provider that implements a controller should also create something called an `APIExportEndpointSlice` alongside their `APIExport`. This will "slice and dice" the list of available virtual workspace endpoints. For example, a [Partition](../sharding/partitions.md) can be provided to restrict provided virtual workspace URLs to a part of the sharded kcp setup.
+An API provider that implements a controller can optionally also create something called an `APIExportEndpointSlice` alongside their `APIExport`. This will "slice and dice" the list of available virtual workspace endpoints. For example, a [Partition](../sharding/partitions.md) can be provided to restrict provided virtual workspace URLs to a part of the sharded kcp setup.
```

Comment:
I would rather change this paragraph to mention what's mentioned below, and eventually have a small box saying prior to kcp v0.28 you had to do this manually.